### PR TITLE
WFLY-15331 Upgrade Infinispan to 13.0.3.Final.

### PR DESCRIPTION
**Pull request by pferraro**
https://issues.redhat.com/browse/WFLY-15331

### CODE REVIEW

**Review by rhusar — CHANGES_REQUESTED** [review]
It appears the latest changes started to make `org.jboss.as.test.clustering.single.web.NonHaWebSessionPersistenceTestCase#testSessionPersistence` fail instead with `java.lang.AssertionError: Session passivation was configured but session was lost after restart. expected:<3> but was:<1>`.

**Comment by pferraro** [comment]
@rhusar This looks ok now.  I had to both re-disable global state and revert back to the SingleFileStore.

File: clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/FileStoreServiceConfigurator.java [hunk 1]
```diff
@@ -74,5 +74,4 @@ public void accept(SingleFileStoreConfigurationBuilder builder) {
         builder.segmented(false)
                 .location(this.pathManager.get().resolveRelativePathEntry(this.relativePath, this.relativeTo))
                 ;
-    }
-}
+    }}
```
Comment (rhusar) [inline clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/FileStoreServiceConfigurator.java:77]: Nitpick: looks like unintended change.

**Review by rhusar — APPROVED** [review]
Looks good.

File: clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/FileStoreServiceConfigurator.java (hunk 1 above)
Comment (pferraro) [reply clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/FileStoreServiceConfigurator.java:77]: yup

Comment (rhusar) [reply clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/FileStoreServiceConfigurator.java:77]: BTW can you join the call? I see you are replying on here so perhaps...

**Comment by darranl** [comment]
Added ready to merge, I believe Rado's review still stands.

**Pull request merged**